ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/pderesources.properties

```diff
@@ -256,6 +256,7 @@ XMLTextChangeListener_editNames_modifyAttribute=Modify "{0}" attribute of "{1}"
 XMLTextChangeListener_editNames_addAttribute=Add "{0}" attribute to "{1}" node
 
 SearchablePluginsManager_createProjectTaskName=Creating {0} project
+SearchablePluginsManager_SaveStates=save states
```

Comment:
Please check messages used and note the new one uses inconsistent style and wording.

Reply:
same as above, removed.